app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, old_file, new_file = command.split()
```

Comment:
The `split()` method without arguments splits a string into a list where each word is a list item. The `split()` method splits the string at whitespaces by default. So, if your command string contains more than one space between 'cp' and 'old_file', or between 'old_file' and 'new_file', this line will return more than three items, and this will cause a ValueError. You need to handle this situation.